### PR TITLE
Create chemicals

### PR DESCRIPTION
**Pull request by wwdaas**

### CODE REVIEW

**Review by mattbeard03 — CHANGES_REQUESTED** [review]
This does not merge with the current database but instead adds a new file and is incorrect. Please merge with current DB, as per ticket instructions

File: next-app/src/db/chemicals [hunk 1]
```diff
@@ -0,0 +1,1399 @@
...
+          },
+          {
+            "id": 3,
+            "name": "Furfural",
+            "description": "A compound that provides almond and caramel notes to whisky. It contributes to both aroma and finish characteristics.",
+            "title": "Furfural"
+          },
+          {
+            "id": 4,
+            "name": "Maltol",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:24]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 2]
```diff
@@ -0,0 +1,1399 @@
...
+          },
+          {
+            "id": 5,
+            "name": "Guaiacol",
+            "description": "A phenolic compound that contributes to smoky, medicinal notes. It's particularly important in peated whiskies.",
+            "title": "Guaiacol"
+          },
+          {
+            "id": 6,
+            "name": "Cresols",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:36]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 3]
```diff
@@ -0,0 +1,1399 @@
...
+          },
+          {
+            "id": 9,
+            "name": "Diacetyl",
+            "description": "A compound that provides buttery notes to whisky, contributing to its rich flavor profile.",
+            "title": "Diacetyl"
+          },
+          {
+            "id": 10,
+            "name": "Hydroxymethylfurfural",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:60]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 4]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A compound that adds bitterness to whisky, contributing to its complex flavor profile.",
+            "title": "Hydroxymethylfurfural"
+          },
+          {
+            "id": 11,
+            "name": "Lignin",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:66]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 5]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A compound that provides woody, smoky character to whisky, derived from oak barrel aging.",
+            "title": "Lignin"
+          },
+          {
+            "id": 12,
+            "name": "Tannins",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:72]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 6]
```diff
@@ -0,0 +1,1399 @@
...
+          },
+          {
+            "id": 13,
+            "name": "Eugenol",
+            "description": "A compound that provides spicy, clove-like flavor to whisky, typically extracted from oak barrels.",
+            "title": "Eugenol"
+          },
+          {
+            "id": 14,
+            "name": "Glucose",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:84]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 7]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A primary sweetening compound in whisky that contributes to its basic sweet taste.",
+            "title": "Glucose"
+          },
+          {
+            "id": 15,
+            "name": "Fructose",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:90]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 8]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A sugar that contributes a more intense sweetness to whisky compared to other sugars.",
+            "title": "Fructose"
+          },
+          {
+            "id": 16,
+            "name": "Sucrose",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:96]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 9]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A common table sugar that provides clean sweetness to whisky.",
+            "title": "Sucrose"
+          },
+          {
+            "id": 17,
+            "name": "Maltose",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:102]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 10]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A sugar that adds depth with a slight malt-like flavor to whisky.",
+            "title": "Maltose"
+          },
+          {
+            "id": 18,
+            "name": "Aspartame",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:108]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 11]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "An artificial sweetener that can be present in trace amounts, much sweeter than sugar.",
+            "title": "Aspartame"
+          },
+          {
+            "id": 19,
+            "name": "Sugar Alcohols",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:114]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 12]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "Compounds like Sorbitol and Xylitol that add sweetness with less caloric impact.",
+            "title": "Sugar Alcohols"
+          },
+          {
+            "id": 20,
+            "name": "Acetic Acid",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:120]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 13]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A compound that enhances complexity with acidity in smoky whiskies.",
+            "title": "Acetic Acid"
+          },
+          {
+            "id": 21,
+            "name": "Cinnamaldehyde",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:126]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 14]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A compound that adds cinnamon and warmth to whisky's spicy character.",
+            "title": "Cinnamaldehyde"
+          },
+          {
+            "id": 22,
+            "name": "Capsaicin",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:132]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 15]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "A compound that provides a burning, peppery sensation in spicy whiskies.",
+            "title": "Capsaicin"
+          },
+          {
+            "id": 23,
+            "name": "Piperine",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:138]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 16]
```diff
@@ -0,0 +1,1399 @@
...
+          },
+          {
+            "id": 25,
+            "name": "Isoamyl Acetate",
+            "description": "An ester that provides a fruity, banana-like flavor to whisky.",
+            "title": "Isoamyl Acetate"
+          },
+          {
+            "id": 26,
+            "name": "Hexyl Acetate",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:156]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 17]
```diff
@@ -0,0 +1,1399 @@
...
+          },
+          {
+            "id": 31,
+            "name": "Octanoic Acid",
+            "description": "An acid that provides buttery, fatty notes to whisky's dry finish.",
+            "title": "Octanoic Acid"
+          },
+          {
+            "id": 32,
+            "name": "Fatty Acids",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:192]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 18]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "Compounds that add richness and enhance mouthfeel in whisky.",
+            "title": "Fatty Acids"
+          },
+          {
+            "id": 33,
+            "name": "Essential Oils",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:198]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 19]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "Compounds that enhance aromatic complexity in whisky.",
+            "title": "Essential Oils"
+          },
+          {
+            "id": 34,
+            "name": "Fatty Acids and Esters",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:204]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 20]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "Compounds that enhance the richness and mouthfeel of whisky's finish.",
+            "title": "Fatty Acids and Esters"
+          },
+          {
+            "id": 35,
+            "name": "Sugars",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:210]: Not in the mock data.

File: next-app/src/db/chemicals [hunk 21]
```diff
@@ -0,0 +1,1399 @@
...
+            "description": "Compounds that contribute to sweetness and texture in whisky.",
+            "title": "Sugars"
+          },
+          {
+            "id": 36,
+            "name": "Sugars and Carbohydrates",
```
Comment (mattbeard03) [inline next-app/src/db/chemicals:216]: Not in the mock data.

**Review by mattbeard03 — CHANGES_REQUESTED** [review]
Why did you put chemicals in that dont exist in the database?

**Comment by wwdaas** [comment]
[db.json](https://github.com/user-attachments/files/17530720/db.json)

**Comment by mattbeard03** [comment]
> [db.json](https://github.com/user-attachments/files/17530720/db.json)

You need to re-commit this merge request please.

**Comment by wwdaas** [comment]
[db.json](https://github.com/user-attachments/files/17532674/db.json)

**Comment by mattbeard03** [comment]
@wwdaas please do the following:

Re-branch.

Open an editor on the new branch from current main.

Edit the JSON file located in the db folder.

Submit a new merge request.

**Pull request closed without merging**